gomi.go

```diff
@@ -0,0 +1,24 @@
+package sample_circleci
+
+import (
+	"errors"
+	"fmt"
+)
+
+func Addd(a, b int) int {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported function Addd should have comment or be unexported